package.cjs.json

```diff
@@ -0,0 +1,11 @@
+{
+  "imports": {
+    "#js/*": "mathjax-full/cjs/*",
+    "#source/*": "mathjax-full/components/cjs/*",
+    "#root/*": "mathjax-full/mjs/components/cjs/*",
```

Comment:
Just FYI, the `mjs` here should be `cjs`, but since you should be removing this file anyway in favor of the one in `components/bin`, there is really no need to fix this.